app/src/main/java/com/openipc/pixelpilot/VideoActivity.java

```diff
@@ -112,6 +112,9 @@ public void run() {
     private ConstraintSet constraintSet;
     private WfbNgLink wfbLink;
 
+    private static final String PREF_DRONE_USERNAME = "drone_username";
```

Comment:
I'd prefer to have it working by default - i.e. having the root/12345 if nothing else was chosen

Reply:
A few days ago I opened Drone Settings, and the WebUI page required me to set a new password, so I set 123456 (not knowing 12345 is the default). I could never open Drone Settings ever since... the root/12345 will still be default, the credentials option is added just in case.